shared/featureconfig/flags.go

```diff
@@ -21,6 +21,10 @@ var (
 	PraterTestnet = &cli.BoolFlag{
 		Name:  "prater",
 		Usage: "Run Prysm configured for the Prater test network",
+	} // PraterTestnet flag for the multiclient eth2 testnet.
```

Comment:
I think this should looks like `L15TestNet flag for the LUKSO eth2 testnet.` here.

Reply:
this line is ok, its for a prater testnet
![Screenshot from 2021-09-23 10-02-08](https://user-images.githubusercontent.com/26545969/134473182-4410e325-a6c5-4b05-9b00-a6f3d6a29465.png)

Reply:
its done twice, I'll erase it